ykman/cli/slot.py

```diff
@@ -252,9 +252,9 @@ def static(ctx, slot, password, no_enter, force):
                 else:
                     break
 
-        password = generate_static_pw(chars)
+        password = generate_static_pw(chars).decode()
```

Comment:
Why no `'utf-8'` argument?

Reply:
Because it's the default value anyway. But I can add it if it makes it more clear / for consistency reasons.